conf/config.toml

```diff
@@ -54,6 +54,12 @@ interval = "15s"
 ## prometheus pushgateway address, leaves it empty will disable prometheus.
 address = ""
 
+[pd-server]
+## the metric storage is the cluster metric storage. This is use for query metric data.
+## Currently we use prometheus as metric storage, we may use PD/TiKV as metric storage later.
+## For usability, recommended to temporarily set it to the prometheus address, eg: http://127.0.0.1:9090
+metric-storage = ""
```

Comment:
It sounds like a kind of storage, but actually it is an address.

Reply:
Ye, I already talk with @lonng , @siddontang, they prefer this. :)

Reply:
how about `metric-storage-url`?

Reply:
The usage of `metric-storage` will not only indicate the `url address`.